pages/resources/index.vue

```diff
@@ -55,21 +55,18 @@ const client = createClient()
 
 const ResourcesSearchResults = () =>
   import('@/components/Resources/ResourcesSearchResults.vue')
-const PlatformSearchResults = () =>
-  import('@/components/Resources/PlatformSearchResults.vue')
-const ToolSearchResults = () =>
-  import('@/components/Resources/ToolSearchResults.vue')
 
 const resourcesResultsComponents = {
```

Comment:
If all tabs are using the same component, there's no need for a dynamic component. Removing this logic will increase performance and redundant code.